spec/pluralrules.html

```diff
@@ -281,106 +281,16 @@ <h1>Properties of Intl.PluralRules Instances</h1>
-         1. Return a new Plural Rules Operands Record { [[Number]]: abs(_n_), [[IntegerDigits]]: _i_, [[FractionDigits]]: _f_, [[NumberOfFractionDigits]]: _fracDigitCount_, [[FractionDigitsWithoutTrailing]]: _significantFrac_, [[NumberOfFractionDigitsWithoutTrailing]]: _significantFracDigitCount_ }.
-       </emu-alg>
-    </emu-clause>
-
-    <emu-clause id="sec-plural-rules-operands-records">
-      <h1>Plural Rules Operands Records</h1>
-
-      <p>
-        Each <dfn id="plural-operands-record">Plural Rules Operands Record</dfn> has the fields defined in <emu-xref href="#table-plural-operands"></emu-xref>.
-      </p>
-
-       <emu-table id="table-plural-operands">
-         <emu-caption>Plural Rules Operands Record Fields</emu-caption>
-         <table class="real-table">
-           <thead>
-             <tr>
-               <th>Field Name</th>
-               <th>Value Type</th>
-               <th>UTS #35 Operand</th>
-               <th>Description</th>
-             </tr>
-           </thead>
-           <tr>
-             <td>[[Number]]</td>
-             <td>Number</td>
-             <td>n</td>
-             <td>Absolute value of the source number</td>
-           </tr>
-           <tr>
-             <td>[[IntegerDigits]]</td>
-             <td>Number</td>
-             <td>i</td>
-             <td>Integer part of [[Number]].</td>
-           </tr>
-           <tr>
-             <td>[[FractionDigits]]</td>
-             <td>Number</td>
-             <td>f</td>
-             <td>Visible fraction digits in [[Number]], <em>with</em> trailing zeroes, as an integer having [[NumberOfFractionDigits]] digits.</td>
-           </tr>
-           <tr>
-             <td>[[NumberOfFractionDigits]]</td>
-             <td>Number</td>
-             <td>v</td>
-             <td>Number of visible fraction digits in [[Number]], <em>with</em> trailing zeroes.</td>
-           </tr>
-           <tr>
-             <td>[[FractionDigitsWithoutTrailing]]</td>
-             <td>Number</td>
-             <td>t</td>
-             <td>Visible fraction digits in [[Number]], <em>without</em> trailing zeroes, as an integer having [[NumberOfFractionDigitsWithoutTrailing]] digits.</td>
-           </tr>
-           <tr>
-             <td>[[NumberOfFractionDigitsWithoutTrailing]]</td>
-             <td>Number</td>
-             <td>w</td>
-             <td>Number of visible fraction digits in [[Number]], <em>without</em> trailing zeroes.</td>
-           </tr>
-         </table>
-       </emu-table>
-    </emu-clause>
-
     <emu-clause id="sec-pluralruleselect" type="implementation-defined abstract operation">
       <h1>
         PluralRuleSelect (
-          _locale_: a String,
-          _type_: *"cardinal"* or *"ordinal"*,
-          _n_: a finite Number,
-          _operands_: a Plural Rules Operands Record derived from formatting _n_,
+          _pluralRules_: an Intl.PluralRules,
+          _s_: a decimal String,
```

Comment:
I think this is going too far; PluralRuleSelect is implementation-defined but can still only act on its input (which should therefore be appropriately scoped and similar to PluralRuleSelectRange).
```suggestion
          _locale_: a language tag,
          _type_: *"cardinal"* or *"ordinal"*,
          _s_: a decimal String,
```